Source/udGeoZone.cpp

```diff
@@ -1042,6 +1042,17 @@ udResult udGeoZone_SetFromSRID(udGeoZone *pZone, int32_t sridCode)
       pZone->latLongBoundMin = udDouble2::create(-90, -180);
       pZone->latLongBoundMax = udDouble2::create(90, 180);
       break;
+    case 4936: // ETRS89
+      pZone->datum = udGZGD_ETRS89;
+      pZone->projection = udGZPT_ECEF;
+      pZone->zone = 0;
+      pZone->scaleFactor = 1.0;
+      pZone->unitMetreScale = 1.0;
+      udStrcpy(pZone->zoneName, "");
+      udGeoZone_SetSpheroid(pZone);
+      pZone->latLongBoundMin = udDouble2::create(34.5, -10.67);
+      pZone->latLongBoundMax = udDouble2::create(71.05, 31.55);
```

Comment:
![image](https://user-images.githubusercontent.com/9654665/143510217-6f4eb712-204c-4816-ac3b-6c0bf6ffee0a.png)

Are these supposed to be the same? Given that its just ECEF you can probably just use

```suggestion
      pZone->latLongBoundMin = udDouble2::create(-90, -180);
      pZone->latLongBoundMax = udDouble2::create(90, 180);
```

Reply:
Current bounds are the bounds specified by spatial reference. There is a small difference with the bounds from epsg.io but I think the bounds are here because the height value from the ellipsoid will be incorrect outside of this bounds.